### PR TITLE
My first capstone project : Expo Book Fair

### PR DESCRIPTION
**Pull request by Debas-31**
# My First Capstone Project: Expo Book Fair
>In this project of my capstone added:
- Semantic HTML tags.
- Best practices in HTML code.
- CSS selectors correctly.
- CSS box model.
- Flexbox to place elements on the page.
- Ability to create UIs adaptable to different screen sizes using media queries.
- GitHub Pages to deploy web pages.
- JavaScript best practices and language style guides in code.
- JavaScript to manipulate DOM elements.
- JavaScript events.
- Objects to store and access data.
- Technical concepts to other technical people.
- [Loom video for my capstone project](https://www.loom.com/share/111e08eeae4b4c51857f41fc7a034ef4)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,56 @@
...
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Debas-31/dbus-first-capstone/issues).
+
+### Show your support
+
+Give a ⭐️ if you like this project!
+
+### Acknowledgments
+- Github
```
Comment (Davidosky007) [inline README.md:52]: * I suggest you acknowledge the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) by [Cindy Shin ](https://www.behance.net/adagio07) as required by the[ Creative Commons License](https://creativecommons.org/licenses/).

File: about.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                    <ul class="nav-down-link">
+                        <li><a class="home-mob" href="index.html">Home</a></li>
+                        <li><a class="about-mob" href="about.html">About</a></li>
+                        <li><a class="close-btn" href="#"><i class="fas fa-times"></i></a></li>
+                    </ul>
+                </div>
+            </nav>
+            <div class="heading-container">
+                <img class="book-icons" src="./Asset/background/additional-background-overlay-icons.png" alt="background image icons">
+                <div class="heading">
```
Comment (Davidosky007) [inline about.html:37]: * Kindly stick to the original design as much as possible as your navbar and header section looks slightly different from the original design, please compare and make the necessary changes 🙏 
## ORIGINAL 
image
## YOURS 
* Note it is required to stick to the Colors,
- Typography: font face, size and weight.
- Layout: composition and space between elements. Content can be different eg( text, title of the website etc)
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @Debas-31 👋, 

Good job so far! 💪 
* No linter errors ✔️ 
* Well written PR ✔️ 
* Descriptive commit messages ✔️ 
* Proper use of git-flow ✔️ 
* Nice video presentation ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



## Required Changes ♻️
_Check the comments under the review._



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @Debas-31  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the requested changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**